CPAC/pipeline/cpac_pipeline.py

```diff
@@ -347,8 +344,8 @@ def run_workflow(sub_dict, c, run, pipeline_timing_info=None, p_name=None,
     Setting MKL_NUM_THREADS to 1
     Setting ANTS/ITK thread usage to {ants_threads}
     Maximum potential number of cores that might be used during this run: {max_cores}
-
-"""
+{random_state}
+"""  # noqa E501
```

Comment:
Pardon my ignorance, but what is `# noqa E501`?

Reply:
`# noqa E501` [tells flake8](https://flake8.pycqa.org/en/4.0.1/user/violations.html#in-line-ignoring-errors) that including lines [> 79 characters](https://www.flake8rules.com/rules/E501.html) in the preceding string is intentional